packages/cf-component-box/README.md

```diff
@@ -0,0 +1,53 @@
+# cf-component-box
+
+> Cloudflare Box Component
+
+This is styling component similar to cf-component-flex which allows for arbitrary styling.
+
+This component should only be used when the built in style doesn't quite fit the bill and when
+it can't be done with fela. Ideally, cf-ui components should have the right styling built in for
+most cases. In the few edge cases that pop up in real world feature development, this is an alternative
+to wrapping your component in a `<div>` or `<span>` and styling it with css to accomodate one-offs.
```

Comment:
Yeah its possible to use div, and thats actually what we're doing now. 

The idea with this is that the fela migration isn't 100% complete yet, and we've all got a bunch of stuff on our plates to either write or refactor. Instead of doing it with div's and either css or inline styles like we've been doing, we'd be using a component that implements all its styling behavior in fela.  It lets us:
   - start using fela today without the migration being complete
   - which lets us use inline styles today with all the perks of fela
   - If we use a single component for these edge cases we can track and replace them easier when the migration is complete.